### PR TITLE
Feature/issue191 wipp library refactor

### PR DESCRIPTION
**Pull request by kannabhargav**

### CODE REVIEW

**Review by sunnielyu — CHANGES_REQUESTED** [review]
Can we separate the ingress from the service?

**Review by mapleknight — CHANGES_REQUESTED** [review]
Please split ingress from other application resources.

**Review by mapleknight — CHANGES_REQUESTED** [review]
you don't need to add services.yaml in prod deployment folder.

File: deploy.sh [hunk 1]
```diff
@@ -14,8 +14,9 @@ rm deploy/kubernetes/frontend-deployment.yaml.bak
...
-    deploy/kubernetes/services.yaml
-rm deploy/kubernetes/services.yaml.bak
+    deploy/kubernetes/ingress.yaml
+rm deploy/kubernetes/ingress.yaml.bak
 
 kubectl apply --kubeconfig=${KUBECONFIG} -f deploy/kubernetes/services.yaml
-kubectl apply --kubeconfig=${KUBECONFIG} -f deploy/kubernetes/frontend-deployment.yaml
\ No newline at end of file
+kubectl apply --kubeconfig=${KUBECONFIG} -f deploy/kubernetes/ingress.yaml
+kubectl apply --kubeconfig=${KUBECONFIG} -f deploy/kubernetes/frontend-deployment.yaml
```
Comment (mapleknight) [inline deploy.sh]: Please change the apply sequence.

**Review by mapleknight — CHANGES_REQUESTED** [review]

File: deploy.sh (hunk 1 above)
Comment (kannabhargav) [reply deploy.sh]: updated the sequence

**Review by sunnielyu — APPROVED** [review]

**Pull request merged**